tests/dialects/test_druid.py

```diff
@@ -0,0 +1,11 @@
+from tests.dialects.test_dialect import Validator
+
+
+class TestDruid(Validator):
+    dialect = "druid"
+
+    def test_druid(self):
+        self.validate_identity("SELECT CEIL(__time TO WEEK) FROM t")
+        self.validate_identity("SELECT CEIL(col) FROM t")
+        self.validate_identity("SELECT FLOOR(__time TO WEEK) FROM t")
+        self.validate_identity("SELECT FLOOR(col) FROM t")
```

Comment:
add a test

validate_all(
 floor(x to week),
write: {
  "": ...
}
)

to make sure transpilation can happen to other dialects, they don't need to be correct but they shouldn't crash